hkato-intern/elearning_app
# User categories page

branch: [user-categories-page]
## Function
- Store data to lessons table if user clicks start button of category.
- Store answers user selected to answers table.

## How to check
- Make sure you logged in as not admin but user.
- Go to categories page.
- Make sure there is at least one question in the category.
- Click start and answer the questions.
- You can see the data in the lessons and answers tables.
